### DISCUSSION TITLE
IB gateway 10.19.2e-stable released

### DISCUSSION THREAD

**Discussion by gnzsnz** [gnzsnz/ib-gateway-docker#31] [Announcements]
IB gateway [10.19.2e-stable](https://github.com/users/gnzsnz/packages/container/ib-gateway/138208999?tag=10.19.2e) has been released and it's available in the container registry.

**Comment by gp-slick-coder** [comment]
Thank you, your version of ib-gateway is working fine. 🥇

**Reply by gnzsnz** [reply]
thanks! enjoy it

I have released a beta version of [TWS with RDP](https://github.com/gnzsnz/ib-gateway-docker/pkgs/container/tws-rdesktop), give it a try!
